phat/artifacts/action_evm_transaction/action_evm_transaction.contract

```diff
@@ -1 +1 @@
"index":1,"name":"CouldNotReadInput"}]}},"path":["ink_primitives","LangError"]}},{"id":7,"type":{"def":{"variant":{"variants":[{"fields":[{"type":8}],"index":0,"name":"Ok"},{"fields":[{"type":6}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":8},{"name":"E","type":6}],"path":["Result"]}},{"id":8,"type":{"def":{"tuple":[9,9,9]}}},{"id":9,"type":{"def":{"primitive":"u16"}}},{"id":10,"type":{"def":{"variant":{"variants":[{"fields":[{"type":0}],"index":0,"name":"Ok"},{"fields":[{"type":6}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":0},{"name":"E","type":6}],"path":["Result"]}},{"id":11,"type":{"def":{"variant":{"variants":[{"fields":[{"type":12}],"index":0,"name":"Ok"},{"fields":[{"type":6}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":12},{"name":"E","type":6}],"path":["Result"]}},{"id":12,"type":{"def":{"variant":{"variants":[{"fields":[{"type":3}],"index":0,"name":"Ok"},{"fields":[{"type":13}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":3},{"name":"E","type":13}],"path":["Result"]}},{"id":13,"type":{"def":{"variant":{"variants":[{"index":0,"name":"BadOrigin"},{"index":1,"name":"NotConfigured"},{"index":2,"name":"BadAbi"},{"fields":[{"type":3,"typeName":"String"}],"index":3,"name":"BadParams"},{"index":4,"name":"BadToAddress"},{"index":5,"name":"BadTransaction"},{"index":6,"name":"FailedToSendTransaction"}]}},"path":["action_evm_transaction","action_evm_transaction","Error"]}},{"id":14,"type":{"def":{"variant":{"variants":[{"fields":[{"type":15}],"index":0,"name":"Ok"},{"fields":[{"type":6}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":15},{"name":"E","type":6}],"path":["Result"]}},{"id":15,"type":{"def":{"variant":{"variants":[{"fields":[{"type":5}],"index":0,"name":"Ok"},{"fields":[{"type":13}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":5},{"name":"E","type":13}],"path":["Result"]}},{"id":16,"type":{"def":{"sequence":{"type":2}}}},{"id":17,"type":{"def":{"sequence":{"type":16}}}},{"id":18,"type":{"def":{"variant":{"variants":[{"fields":[{"type":19}],"index":0,"name":"Ok"},{"fields":[{"type":6}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":19},{"name":"E","type":6}],"path":["Result"]}},{"id":19,"type":{"def":{"variant":{"variants":[{"fields":[{"type":16}],"index":0,"name":"Ok"},{"fields":[{"type":13}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":16},{"name":"E","type":13}],"path":["Result"]}},{"id":20,"type":{"def":{"variant":{"variants":[{"fields":[{"type":21}],"index":0,"name":"Ok"},{"fields":[{"type":6}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":21},{"name":"E","type":6}],"path":["Result"]}},{"id":21,"type":{"def":{"variant":{"variants":[{"fields":[{"type":22}],"index":0,"name":"Ok"},{"fields":[{"type":13}],"index":1,"name":"Err"}]}},"params":[{"name":"T","type":22},{"name":"E","type":13}],"path":["Result"]}},{"id":22,"type":{"def":{"composite":{"fields":[{"type":1,"typeName":"[u8; 32]"}]}},"path":["primitive_types","H256"]}}],"version":"4"}
```

Comment:
It's a bad practice for me to use artifacts as releases in this repo. 
Can we setup the Github action to support real witnessed release?

Reply:
Let's make another PR to do this.

Reply:
Ok, I create an issue for this #15 .